docs/advanced/document-store/opensearch.md

```diff
@@ -0,0 +1,414 @@
+(opensearch)=
+
+# Opensearch
+
+One can use [Opensearch](https://opensearch.org/) as the document store for DocumentArray. It is useful when one wants to have faster Document retrieval on embeddings, i.e. `.match()`, `.find()`.
```

Comment:
Could you change from "one" to "you" throughout the doc please? To follow our updated style guidelines (updated like yesterday - they're not public yet)